wdtk-wikibaseapi/src/main/java/org/wikidata/wdtk/wikibaseapi/EditingResult.java

```diff
@@ -41,11 +42,11 @@ public EditingResult(long revisionId) {
     }
 
     /**
-     * The identifier of the revision of the last edit made by the editing action.
-     * If no edit was made, 0L is returned.
+     * The identifier of the revision of the last edit made by the editing action,
+     * if any edit was made.
      */
-    public long getLastRevisionId() {
-        return revisionId;
+    public Optional<Long> getLastRevisionId() {
+        return revisionId == 0 ? Optional.empty() : Optional.of(revisionId);
```

Comment:
Why not use [`OptionalLong`](https://docs.oracle.com/javase/8/docs/api/java/util/OptionalLong.html)?

Reply:
Simply wasn't aware of its existence!